views/pages/ResetPasswordPage.jsx

```diff
@@ -0,0 +1,261 @@
+                    autoFocus={true}
+                    required={true}
+                    fullWidth={true}
+                    margin="normal"
+                    error={
+                      this.state.unInitialized &&
+                      this.state.username.length === 0
+                        ? true
+                        : false
+                    }
+                  />
+                  <br />
+                  <TextField
+                    id="password"
+                    name="password"
+                    type="password"
+                    value={this.state.password}
+                    onChange={this.handleChange('password')}
+                    label="New Password"
+                    required={true}
+                    margin="normal"
+                    fullWidth={true}
+                    error={this.state.unmatched}
+                  />
+                  <br />
+                  <TextField
+                    id="confirmpw"
+                    name="confirmpw"
+                    type="password"
+                    value={this.state.confirmpw}
+                    onChange={this.confirmPassword}
+                    label="Confirm Password"
+                    required={true}
+                    margin="normal"
+                    fullWidth={true}
+                    error={this.state.unmatched}
+                    disabled={this.state.password.length === 0 ? true : false}
+                  />
+                  <br />
+                  <TextField
+                    id="reset_key"
+                    name="reset_key"
+                    type="text"
+                    label="Reset Key"
+                    value={this.state.resetkey}
+                    onChange={this.handleChange('resetkey')}
+                    required={true}
+                    fullWidth={true}
+                    margin="normal"
+                    error={
+                      this.state.rkInitialized &&
+                      this.state.resetkey.length === 0
+                        ? true
+                        : false
+                    }
+                  />
+                </div>
+              </div>
+            </div>
+            {this.state.windowWidth < 620 ? null : (
+              <CardMedia
+                style={{
+                  width: '250px',
+                  margin: '50px',
+                  backgroundImage: `url("${basePath}/img/dpdash.png")`,
+                  backgroundSize: 'contain',
+                  backgroundRepeat: 'no-repeat',
+                  backgroundPosition: 'center center',
+                }}
+                title="DPdash"
+              />
+            )}
+          </div>
+          <div
+            style={{
+              padding: '12px',
+              float: 'right',
+            }}
+          >
+            <Button
+              color="primary"
+              onClick={() => this.openWindow(`${basePath}/login`)}
```

Comment:
should this be a router link with a route constant?

Reply:
good catch [c247246](https://github.com/AMP-SCZ/dpdash/pull/319/commits/c2472461f08e7bd81bf1909aa22afd4c81658089)